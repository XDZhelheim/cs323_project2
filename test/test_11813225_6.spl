struct cui {
  int cuiMax;
  float cuiPro;
  char cuiMini;
};

int cuicui()
{
  struct cui cuiAir;
  cuiAir.cui1 = 100;
}