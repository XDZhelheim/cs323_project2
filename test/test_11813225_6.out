Error type 14 at Line 10: accessing an undefined structure member
Error type 5 at Line 10: unmatching types appear at both sides of the assignment operator
